app/views/cases/amendments/new.html.slim

```diff
@@ -26,4 +26,4 @@
   = f.text_area :request_amends_comment
 
   .button-holder
-    =  f.submit "Request amends" ,  {class: "button", role:"button"}
+    =  f.submit t('button.request_amends') ,  {class: "button", role:"button"}
```

Comment:
While you're making changes might as well make the second bit prettier:
`{ class: 'button', role: 'button' }`